src/system_data.jl

```diff
@@ -200,14 +200,13 @@ function remove_time_series!(
     ::Type{T},
     data::SystemData,
     component::InfrastructureSystemsComponent,
-    initial_time::Dates.DateTime,
     name::String,
 ) where {T <: TimeSeriesData}
     type = time_series_data_to_metadata(T)
-    time_series = get_time_series(type, component, initial_time, name)
+    time_series = get_time_series(type, component, name)
     uuid = get_time_series_uuid(time_series)
     # TODO: can this be atomic?
```

Comment:
Is the comment still valid?

Reply:
It's hopeful but can be removed if I can't think of anything before we merge to master.